maps/submaps/surface_submaps/mountains/CrashedMedShuttle1.dmm

```diff
@@ -79,10 +79,10 @@
 "bA" = (/obj/structure/girder/displaced,/turf/template_noop,/area/submap/CrashedMedShuttle)
 "bB" = (/obj/structure/shuttle/engine/propulsion{tag = "icon-propulsion_r (WEST)"; icon_state = "propulsion_r"; dir = 8},/turf/simulated/floor/outdoors/rocks,/area/submap/CrashedMedShuttle)
 "bC" = (/obj/structure/shuttle/engine/heater{icon_state = "heater"; dir = 8},/obj/structure/window/reinforced{dir = 4},/turf/simulated/mineral/floor/ignore_mapgen,/area/submap/CrashedMedShuttle)
-"bD" = (/obj/structure/table/standard,/turf/simulated/shuttle/floor/purple,/area/submap/CrashedMedShuttle)
-"bE" = (/obj/structure/table/standard,/obj/item/weapon/storage/fancy/vials,/turf/simulated/shuttle/floor/purple,/area/submap/CrashedMedShuttle)
+"bD" = (/obj/structure/table/standard,/obj/item/weapon/storage/firstaid/adv,/turf/simulated/shuttle/floor/purple,/area/submap/CrashedMedShuttle)
```

Comment:
Better, but still not actually worth going out of your way to follow the signal for, and definitely not worth the risk posed by the dozen+ (fourteen, last I counted?) Actual Dangerous Spiders. Plus it removes the vials, so... not really that much better, but.